qcelemental/physical_constants/ureg.py

```diff
@@ -28,33 +28,53 @@ def build_units_registry(context):
     ureg.define(
         "hartree_inverse_meter = {} / hartree / m".format(phys_const["hartree-inverse meter relationship"]["value"])
     )
+    ureg.define("plancks_constant = {} * joule * s".format(phys_const["planck constant"]["value"]))
 
     # Energy
-    ureg.define("hartree = {} * joule = E_h = hartree_energy".format(phys_const["hartree energy"]["value"]))
+    ureg.define("hartree = {} * joule = E_h = hartree_energy = au_energy".format(phys_const["hartree energy"]["value"]))
     ureg.define("electron_volt = {} * J = eV".format(phys_const["electron volt-joule relationship"]["value"]))
 
-    # Constants
-    ureg.define("electron_mass = {} * kg".format(phys_const["electron mass"]["value"]))
-    ureg.define("elementary_charge = {} * coulomb = e".format(phys_const["elementary charge"]["value"]))
-    ureg.define("plancks_constant = {} * joule * s".format(phys_const["planck constant"]["value"]))
-
-    # Distance
-    ureg.define("bohr = {} * meter = bohr_radius = Bohr".format(phys_const["bohr radius"]["value"]))
-    ureg.define("wavenumber = 1 / centimeter")
-    ureg.define("Angstrom = angstrom")
-
-    # Time
-    ureg.define(f"au_time = {phys_const['atomic unit of time']['value']} * {phys_const['atomic unit of time']['unit']}")
-
-    # Masses
+    # Mass
+    ureg.define("electron_mass = {} * kg = au_mass".format(phys_const["electron mass"]["value"]))
     ureg.define(
         "atomic_mass_unit = {} * kilogram = u = amu = dalton = Da".format(phys_const["atomic mass constant"]["value"])
     )
 
-    # Coulomb
+    # Charge
+    ureg.define("elementary_charge = {} * coulomb = e = au_charge".format(phys_const["elementary charge"]["value"]))
     ureg.define("statcoulomb = coulomb / 2997924580 = statC")
     ureg.define("debye = 1e-18 * statcoulomb * cm = D")
 
+    # Distance
+    ureg.define("bohr = {} * meter = bohr_radius = Bohr = au_length".format(phys_const["bohr radius"]["value"]))
```

Comment:
What does the following look like? `str(ureg.Unit("5.0 * au_length"))`

Reply:
```
In [5]: str(qcelemental.constants.ureg.Unit("5.0 * au_length"))                                                                                                                    
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-5-e84761374df7> in <module>
----> 1 str(qcelemental.constants.ureg.Unit("5.0 * au_length"))

~/anaconda3/envs/qcelemental/lib/python3.7/site-packages/pint/unit.py in __new__(cls, units)
     44             inst._units = units
     45         elif isinstance(units, string_types):
---> 46             inst._units = inst._REGISTRY.parse_units(units)._units
     47         elif isinstance(units, _Unit):
     48             inst._units = units._units

~/anaconda3/envs/qcelemental/lib/python3.7/site-packages/pint/registry.py in parse_units(self, input_string, as_delta)
    790             :class:`ValueError` if the expression is invalid.
    791         """
--> 792         units = self._parse_units(input_string, as_delta)
    793         return self.Unit(units)
    794 

~/anaconda3/envs/qcelemental/lib/python3.7/site-packages/pint/registry.py in _parse_units(self, input_string, as_delta)
    900             as_delta = self.default_as_delta
    901 
--> 902         return super(NonMultiplicativeRegistry, self)._parse_units(input_string, as_delta)
    903 
    904     def _define(self, definition):

~/anaconda3/envs/qcelemental/lib/python3.7/site-packages/pint/registry.py in _parse_units(self, input_string, as_delta)
    810         units = ParserHelper.from_string(input_string)
    811         if units.scale != 1:
--> 812             raise ValueError('Unit expression cannot have a scaling factor.')
    813 
    814         ret = {}

ValueError: Unit expression cannot have a scaling factor.
```

```
In [6]: str(qcelemental.constants.ureg.Unit("au_length"))                                                                                                                          
Out[6]: 'bohr'
```

Reply:
That wasn't what I was worried about. But is concerning. Let me know if I need to play around with this.

Reply:
I'm not sure I see the issue. Do you want `str(ureg.Unit("au_length")) == "au_length"`? Is it something more?

Reply:
I see, the first is just my typo. Got it. This is fine.

Reply:
I have attempted to fix this with the latest commit (2624d4f), with testing. This does not pass, but looking at why is instructive:

1. some units like "m" are converted to "meter"
2. the amu/kg conversion breaks for some reason

Reply:
> I see, the first is just my typo. Got it. This is fine.

In that case, we should probably revert 2624d4f.